packages/react/src/useList.ts

```diff
@@ -56,6 +57,12 @@ export const useList = <
   const [direction, setDirection] = useState<"forward" | "backward">("forward");
   const clearCursor = useCallback(() => setCursor(undefined), []);
 
+  useMemo(() => {
```

Comment:
My first instinct to do something like this would be to use a `useEffect`. What goes into the decision of `useEffect` vs `useMemo`

Reply:
just a 🧠 fart -- thought i was typing useEffect